wpimath/src/main/java/edu/wpi/first/math/system/NumericalIntegration.java

```diff
@@ -45,9 +44,8 @@ public static double rk4(DoubleFunction<Double> f, double x, double dtSeconds) {
    * @param dtSeconds The time over which to integrate.
    * @return The result of Runge Kutta integration (4th order).
    */
-  @SuppressWarnings("overloads")
   public static double rk4(
-      BiFunction<Double, Double, Double> f, double x, Double u, double dtSeconds) {
+      BiFunction<Double, Double, Double> f, double x, double u, double dtSeconds) {
```

Comment:
`f` could be a `DoubleBinaryOperator` and avoid more boxing, though it would break anyone who stores it in a variable or return type

https://docs.oracle.com/javase/8/docs/api/java/util/function/DoubleBinaryOperator.html